wazo_calld/plugins/calls/services.py

```diff
@@ -492,6 +498,37 @@ def set_answered_time(self, channel_id):
             logger.debug('channel %s not found', channel_id)
             raise NoSuchCall(channel_id)
 
+    @staticmethod
+    def conversation_direction_from_channels(ari, channels):
+        all_directions = []
+
+        for channel in channels:
+            try:
+                call_direction = (
+                    ari.channels.getChannelVar(channelId=channel.id, variable='WAZO_CALL_DIRECTION')['value']
+                )
+            except ARINotFound:
+                continue
+            else:
+                if call_direction:
+                    all_directions.append(call_direction)
+
+        return CallsService._conversation_direction_from_directions(all_directions)
+
+    @staticmethod
+    def _conversation_direction_from_directions(directions):
+        if 'outbound' in directions and 'inbound' in directions:
+            return 'unknown'
+
+        if 'outbound' in directions:
+            return 'outbound'
+
+        if 'inbound' in directions:
+            return 'inbound'
+
+        # NOTE(afournier): internal should have the lowest priority
```

Comment:
I don't think this comment very relevant. I think the code is explicit enough. If you have a specific reason for the comment, please add it. Else, I think we can remove this comment.

Reply:
I will remove it.